pkg/app/pagewriter/sign_in_page.go

```diff
@@ -104,6 +105,11 @@ func loadCustomLogo(logoPath string) (string, error) {
 		return "", nil
 	}
 
+	if strings.HasPrefix(logoPath, "https://") {
+		// Return img tag pointing to the URL.
+		return fmt.Sprintf("<img src=\"%s\" alt=\"Logo\" />", logoPath), nil
```

Comment:
Not blocking but a tip for the future, instead of `\"%s\"` you can just do `%q` and it has the same effect :slightly_smiling_face: